contrail-ui/src/App.tsx

```diff
@@ -2,12 +2,18 @@ import React from "react";
 import { BrowserRouter as Router, Redirect, Route } from "react-router-dom";
 import Auth from "./components/auth/auth-page/Auth";
 import MainFrame from "./components/main/MainFrame";
+import AuthorizedRoute from "./HOCs/AuthorizedRoute";
+import * as ROUTES from "./routes";
 
 function App() {
   return (
     <Router>
-      <Route path="/auth" exact={true} component={Auth} />
-      <Route path="/main" exact={true} component={MainFrame} />
+      <Route path={ROUTES.AUTH} component={Auth} />
+      <AuthorizedRoute path={ROUTES.MAIN} component={MainFrame} />
+      <AuthorizedRoute path={ROUTES.FILES} component={MainFrame} authRoute={"files"} />
```

Comment:
Does this mean for every single authorized route we have to add it here?

Reply:
isn't there like a children route option in react?

Reply:
Yeah we can edit this later when we have the children components ready. We just move these into those components and only have /auth and /app here